src/smooth.h

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Maybe we should have a src/drawing directory where drawing.[ch] and smooth.[ch] reside? Maybe rename drawing.[ch] to something more fitting? Or just move all the stuff in smooth.[ch] to drawing.[ch]?

Reply:
Maybe it is better to restrict drawing.[ch] to the actual drawing operations (there could be more in the future).

smooth.[ch] is really about manipulations of `coordlist` before drawing.

What about coordlist_ops.[ch]?

Reply:
Sounds good! I think there are functions from drawing going there then? If so, we could move them in a follow-up PR.

Reply:
yes coord_list_prepend and coord_list_free would go there...